scripts/tap-generate-docs/src/tap_generate_docs.clj

```diff
@@ -36,7 +36,9 @@
   [schema properties]
   {:pre [(not (contains? properties "type"))]}
   (sort-by #(get % "name")
-           (map (partial convert-multiary-type schema) properties)))
+           (reduce (fn [acc p] (concat acc (convert-multiary-type schema p)))
+                   []
+                   properties)))
```

Comment:
This is very odd to me. `map` produces a seq so reducing into a `vector` seems weird. Was the problem that it wasn't being realized? `doall` should work there.

Reply:
The reason for the reduce is that we pulled the literal vector of `[(convert-multiary-type ...)]` into the return value of `convert-multiary-type`, so it's reducing into a vector because that's the return value's structure.